end_to_end/3-mitigate-bias-train-model2-registry-e2e.ipynb

```diff
@@ -0,0 +1,1340 @@
+{
```

Comment:
<p>The object exists in <code>training_job_2_info['HyperParameters']['sagemaker_submit_directory']</code>  in what circumstance would <code>list_response</code>  be empty. The code runs, but it not have the intended behavior</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1974/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>good question. doing defensive programming to list resources and if they exist use them, if not, create them.</p>